core/src/main/java/org/mule/api/config/MuleProperties.java

```diff
@@ -203,4 +203,5 @@
     public static final String MULE_FAIL_IF_DELETE_OPEN_FILE = SYSTEM_PROPERTY_PREFIX + "failIfDeleteOpenFile";
     public static final String CONTENT_TYPE_PROPERTY = "Content-Type";
     public static final String DISABLE_ERROR_COUNT_ON_ERROR_NOTIFICATION_DISABLED = SYSTEM_PROPERTY_PREFIX + "disable.error.count.on.error.notifications.disabled";
+    public static final String MULE_DEFAULT_BOOLEAN_VALUE = SYSTEM_PROPERTY_PREFIX + "default.boolean.value";
```

Comment:
For mule-3.x (3.10.0) add the actual parameter, you can then send a PR to 3.9.x adding the system property instead, though I think we need to make its name more specific to where it is actually used.